src/Datadog.Trace/SpanContext.cs

```diff
@@ -78,6 +84,11 @@ private SpanContext(ulong? traceId, string serviceName)
         /// </summary>
         public string ServiceName { get; set; }
 
+        /// <summary>
+        /// Gets the origin of the trace
+        /// </summary>
+        public string Origin { get; }
```

Comment:
According to the specs:
```_dd.origin is a special tag that is used then in our backend to change the sampling logic. This tag is not visible in our UI.```

I don't think we should expose it in a public API

Reply:
Ok, I'll change it to `internal`